readme.md

```diff
@@ -46,16 +46,16 @@ You will need to find the full path for the file e.g.
 Manually add the ScalaClean plugin to your project definition(s) - this will be automated later:
 
 ```scala
-.settings(
+settings(
   addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "4.3.0" cross CrossVersion.full),
   scalacOptions ++= {
     val srcLocations = (sourceDirectories in Compile).value.mkString(java.io.File.pathSeparator)
     Seq(
       "-Yrangepos", // from semanticdb instructions
       "-Xplugin:/workspace/ScalaClean/analysisPlugin/target/scala-2.12/analysisPlugin_2.12.10-0.1.0-SNAPSHOT-assembly.jar",
-      s"-P:scalaclean-analysis-plugin:srcdirs:${srcLocations}",
+      s"-P:scalaclean-analysis-plugin:srcdirs:$srcLocations"
     )
-  },
+  }
```

Comment:
boo! 😛

Reply:
I'm still horribly torn on training commas

Reply:
I feel comfortable in the "trailingCommas = preserve" + "boy scout it next diff" realm.